governance/votes/Executive vote - January 14, 2022.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Template - [Executive Vote] Recover DAI from Optimism Escrow, Changes to Uniswap LP Vault Debt Floor Parameters, Delegate Compensation - January 14, 2022
+summary: Recover DAI from Optimism escrow, Uniswap LP Vault Debt Floor parameter changes, Recognized Delegate compensation distribution for December.
+date: 2022-01-14T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Recover DAI from Optimism Escrow, Changes to Uniswap LP Vault Debt Floor Parameters, Delegate Compensation - January 14, 2022
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- 10,000,000 DAI will be transferred from the Optimisim Escrow to a user who accidentally burned DAI on the Optimism chain.
+- [Debt Floor (`dust`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-floor) increases across multiple Uniswap LP Vault types, listed in detail below.
+- A total of 48,290 DAI will be transferred to seven Recognized Delegates as part of the [Delegate Compensation Trial](https://vote.makerdao.com/polling/QmbvuhYH?network=mainnet#poll-detail).
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Recover DAI from Optimism Escrow
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/Qmcfb72e), **10,000,000 DAI** will be transferred to forum user lost_some_dai at the address [0xc9b48B787141595156d9a7aca4BC7De1Ca7b5eF6](https://etherscan.io/address/0xc9b48B787141595156d9a7aca4BC7De1Ca7b5eF6) if this executive proposal passes.
```

Comment:
10,000,000 DAI -> 10 million DAI

Reply:
Worth some @, quotations or emphasis on the username? Link to forum account? etc?

Reply:
Changed both and added comment by PE about how changes DAI transfer will happen